File: Java Spring Boot Project/misc/nport_project/MtCProject1/src/main/java/com/nikosportolos/MtCProject1/models/responses/AllBusinessUnitsResponse.java
package com.nikosportolos.MtCProject1.models.responses;

import com.nikosportolos.MtCProject1.models.BusinessUnit;
import java.util.List;

public class AllBusinessUnitsResponse {

    private List<BusinessUnitResponse> businessUnits ;

    public AllBusinessUnitsResponse(List<BusinessUnitResponse> businessUnits) {
        this.businessUnits = businessUnits;
    }

    public List<BusinessUnitResponse> getBusinessUnits() {
        return businessUnits;
    }

    public void setBusinessUnits(List<BusinessUnitResponse> businessUnits) {
        this.businessUnits = businessUnits;
    }
}

File: Java Spring Boot Project/company_directory_app/lib/models/responses/all_business_units_response.dart
import 'package:company_directory_app/models/responses/business_unit_response.dart';

class AllBusinessUnitsResponse {
  final List<BusinessUnitResponse> businessUnits;

  AllBusinessUnitsResponse({this.businessUnits});

  factory AllBusinessUnitsResponse.fromJson(Map<String, dynamic> json) {
    var list = json['businessUnits'] as List;
    List<BusinessUnitResponse> itemsList = list.map((i) => BusinessUnitResponse.fromJson(i)).toList();

    return AllBusinessUnitsResponse(
      businessUnits: itemsList,
    );
  }
}
